back_end/src/test/resources/application.properties

```diff
@@ -11,29 +11,23 @@ spring.jpa.hibernate.ddl-auto=update
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.H2Dialect
 spring.jpa.defer-datasource-initialization=true
-
 # Google Properties
 spring.security.oauth2.client.registration.google.client-id=915852862463-lhg3fcm3q472ka2ls3ijqu86pnco6q24.apps.googleusercontent.com
 spring.security.oauth2.client.registration.google.client-secret=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
 #spring.security.oauth2.client.registration.google.redirect-uri={baseUrl}/login/oauth2/code/{registrationId}
 spring.security.oauth2.client.registration.google.scope=email,profile,openid
 spring.security.oauth2.client.provider.google.user-info-uri=https://www.googleapis.com/oauth2/v3/userinfo
-
 spring.flyway.enabled=false
-
 ## Format the SQL
 spring.jpa.properties.hibernate.format_sql=true
 spring.jpa.properties.hibernate.show_sql=false
-
 # email constants
 grey.from=greynoreply@gmail.com
 grey.password=xkjhaauagkxaqhhp
```

Comment:
please remove these credentials and change it after removal 
instead place them in environment variables or gitignore this file